paddle/fluid/framework/details/build_strategy.h

```diff
@@ -88,7 +88,7 @@ struct BuildStrategy {
   bool fuse_elewise_add_act_ops_{false};
   // Fuse_all_optimizer_ops and fuse_all_reduce_ops require that gradients
   // should not be sparse types
-  bool fuse_all_optimizer_ops_{false};
+  bool fuse_all_optimizer_ops_{true};
```

Comment:
As you say, there are some differences for CPU, I don't think it is suitable to set the default value to true.

Reply:
This effect is very small, and I have run the models of CE and CI, all the tests are passed.

Reply:
If we set `fuse_all_optimizer_ops_` default true, how much will the performance speedup?

Reply:
About 5%, in the transformer base model.

Reply:
This will make pyramid_dnn training process fail without any error.